src/components/AddressSearch/index.tsx

```diff
@@ -224,6 +233,30 @@ function AddressSearch(
 
         onPress?.(values);
     };
+    const selectFocusedOption = () => {
+        if (!resultRef.current) {
+            return;
+        }
+        const data = resultRef?.current[focusedIndex];
+        if (!data) {
+            return;
+        }
+        saveLocationDetails(data, data);
+        setIsTyping(false);
+
+        // After we select an option, we set displayListViewBorder to false to prevent UI flickering
+        setDisplayListViewBorder(false);
+        setIsFocused(false);
+
+        // Clear location error code after address is selected
+        setLocationErrorCode(null);
+    };
+    useKeyboardShortcut(CONST.KEYBOARD_SHORTCUTS.ENTER, selectFocusedOption, {
+        captureOnInputs: true,
```

Comment:
Most of these config values are the default. The only one that's different is `shouldBubble`. Can you first try removing this config object entirely and see if that works? If not, let's pare this back to include only `shouldBubble: true`.